src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
I think `counterparty` is just a more accurate name for `issuer`, emphasizing the symmetry between the two sides.